Cargo.toml

```diff
@@ -159,7 +159,7 @@ wasm-instrument = "0.4.0"
 wasmer = { package = "linera-wasmer", version = "4.3.6-linera.3", default-features = false }
 wasmer-compiler-singlepass = { package = "linera-wasmer-compiler-singlepass", version = "4.3.6-linera.3" }
 wasmparser = "0.101.1"
-wasmtime = "24.0.0"
+wasmtime = { git = "https://github.com/bytecodealliance/wasmtime", rev = "58f82587462543e912b4edcbb247842711ba6415" }
```

Comment:
What happened?

Reply:
The fix we need hasn't been released yet. I added a TODO to update this once a release is available (they tend to release often, so hopefully soon).